### PR TITLE
Add more zealot metadata

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208057678750035

### CODE REVIEW

File: scripts/deploy.ts [hunk 1]
```diff
@@ -435,10 +435,17 @@ function buildCommonPost(buildObj: BuildObj) {
   }
 
   if (zealotApiToken != null && zealotUrl != null && zealotChannelKey != null && !maestroBuild) {
+    const branch = encodeURIComponent(buildObj.repoBranch)
+    const gitCommit = encodeURIComponent(buildObj.guiHash)
+    chdir(buildObj.guiDir)
+    const changes = rmNewline(cmd(`git diff HEAD^ HEAD CHANGELOG.md | grep '^+[^+]'`))
```
[the remark this answers is not in the document]
Comment (samholmes) [reply scripts/deploy.ts:441]: Why is it only matching line removals and not line additions? I'm a little confused on what this regex does.

**Review by samholmes — CHANGES_REQUESTED** [review]
Mainly need clarity on the regex.

File: scripts/deploy.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (paullinator) [reply scripts/deploy.ts:441]: It finds the line additions, not removals. the first ^+ matches the beginning of the line that starts with a `+`. the `[^+]` matches a second character which is anything but a `+`. This prevents matching the +++ at the top of a diff so the only lines that get matched start with a `+` followed by the line added which for our format always starts with a `-` such as `- added: some feature`.

**Review by samholmes — APPROVED** [review]

File: scripts/deploy.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (samholmes) [reply scripts/deploy.ts:441]: Wait, shouldn’t it be ^\+ then?

**Pull request merged**